.github/workflows/ci.yml

```
@@ -0,0 +1,26 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  tests:
+    name: Tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.7.13', '3.8.12', '3.9.12', '3.10.4', '3.11.0', '3.12.0']
```

Would this work? I feel like it would be better to not latch on to minor versions if possible
```suggestion
        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11', '3.12']
```

---

I was going by the listed available versions for python and ubuntu 22.04

---

I think it would be more ideal to test against the major versions if possible.

---

Okay cool that worked